src/lib/Zikula/Bundle/CoreBundle/EventListener/ThemeListener.php

```diff
@@ -95,23 +97,29 @@ public function setDefaultPageAssets(GetResponseEvent $event)
             return;
         }
         $basePath = $event->getRequest()->getBasePath();
+        $jquery = $this->env == 'prod' ? 'jquery.min.js' : 'jquery.js';
```

Comment:
Good point. Maybe `debug` (or even better dev env + debug) would be more appropriate in this case.